### DISCUSSION TITLE
duplicar botão Whatsapp

### DISCUSSION THREAD

**Discussion by SuperPorte** [RamonSilva20/mapos#1624] [Q&A]
Bom dia Galera. Eu queria duplicar o botão whatsapp das OSs, e trocar o parâmetro WEB para API.
Como faço isso?

**Comment by SuperPorte** [comment]
Pelo menos mudar o parâmetro de WEB para API no link do botão.

**Reply by bruno-esilva** [reply]
Também não manjo de programação, mas se for na parte de visualizar a OS (isso na última versão do Mapos) onde tem aquele botão de enviar, vc pode editar o arquivo deste caminho:
application/view/os/visualizaros.php
Na linha 27 fica o botão do whatsapp. Creio que baste copiar o código dessa linha, e colar abaixo, apenas substituindo o web.whatsapp.com por api.whatsapp.com.
Isso já vai duplicar o botão, e vc pode alterar ao final da linha, onde tem o Whatsapp</a> e colocar WEB e API na frente de cada um para facilitar a identificação.
